.eslintrc

```diff
@@ -1,10 +1,15 @@
 {
     "parser": "babel-eslint",
     "extends": ["standard", "standard-react"],
+    "globals": {
+        "__DEV__": false
+    },
     "plugins": [
         "flow-vars"
     ],
     "rules": {
+        "flow-vars/define-flow-type": 1,
+        "flow-vars/use-flow-type": 1,
```

Comment:
it wans't needed before but is now for the plugin to work, it fixes missing reactElement and etc